src/Integration.Vsix/CFamily/PortedFromJava/Protocol.cs

```diff
@@ -140,42 +140,21 @@ private static void Write(BinaryWriter writer, string[] strings)
         /**
            * This method does not close the provided stream.
            */
-           // Optimisation added in C#: filter files during load, rather than loading everything then filtering.
-           // Example: analysing a small C# file produced 1300 issues; ten in the cpp file, the rest in
-           // header files (we don't currently show issues in header files).
-           // The issueFilePath is optional.
-        public static void Read(BinaryReader reader, Action<Message> handleIssue, string issueFilePath = null)
```

Comment:
The `issueFilePath` is no longer used so it doesn't need to be passed by the `CLangAnalyzer`.